### ISSUE TITLE
Recording species counts

### ISSUE THREAD

**Issue by phylojenie** [DataDrivenEcologicalSynthesis/VirtualBiogeography#37]
@superornitho So just to clarify, we are only recording the _first_ screenshot that a particular species appears in, correct? So if Nemo appears in SS_01-01-01_GBR, we record that screenshot as his only occurrence for the GBR region?

But for example if there are two different green crabs in two different screenshots, we would record two occurrences of that species and list the two different screenshots we saw them in?

I'd like to get away from loading the spreadsheet with multiple screenshot names per column and instead record numbers and location data for easier analysis. So for example we could have:

Location | GBR | EAC | T
---------------- | ------ | ------ | ----
Species X | 3 | 1 | 0

Hope this makes sense, let me know what you think. As it stands the data sheet will be very hard to parse because we're not separating the observations by location or recording the count numbers.

**Comment by Crataegus9** [comment]
Yes, that is a better idea, I will follow this structure from now on. I was recording every species in every screenshot in the prospect that we would like to record species interactions, but I think this will take too much work and we won’t easily be able to assess the validity of these interactions. The literature on this subject is quite scarce I believe. 
Have a nice day, 
Étienne

**Comment by phylojenie** [comment]
Good point about the species interactions, however I think we can use our defined locations (Great Barrier Reef, Ocean Trench) etc. as a stand-in for counting species interactions. For example, if a clownfish and a blue tang are both recorded in the GBR screenshots, we can count them as interacting.

I will update the spreadsheet with the recording system I am using and we can both fill it out using this system.

**Comment by Crataegus9** [comment]
Yep, totally agree on that!

**Comment by phylojenie** [comment]
@superornitho Take a look at the new spreadsheet format when you have a chance. I have added a column for each location, and we can record the total number of each species at a given location. If you like this format, we'll go ahead with it!

**Comment by Crataegus9** [comment]
Hi all, 
I don’t see how counting the number of occurence in each « biome » would be useful in our analysis. Also, I don’t want anyone to try counting how many mackerels there are in some scenes.
I would consider something like:

LOCATION         GBR                                                EAC …
Species1           Screenshot#of first occurence            [blank if species not present]
Species2         [blank if species not present]              Screenshot#of first occurence

**Comment by phylojenie** [comment]
I see your point, however tracking the occurrences at each point will give us a better idea of how to match the species to their range maps; for example if one species is observed at both the GBR and in the Sydney Harbour, we could then check to see if their range extends that far in real life.

Also, estimating counts of each species was going to be one of the ways we checked for the accuracy of the abundance of each species.

**Comment by Crataegus9** [comment]
Sorry if my wording was unclear.
 I agree with you on this first point  "tracking the occurrences at each point will give us a better idea of how to match the species to their range maps; for example if one species is observed at both the GBR and in the Sydney Harbour, we could then check to see if their range extends that far in real life."

As for calculating the abundance of each species in each area, I pondered how this could be useful. We will maybe get 1-5 individuals of each species in the whole film (+ some outliers like the mackerels), so I don’t think extrapolating these abundances to this whole part of the ocean makes a lot of sense. Also, I am not sure if GBIF occurrences can offer an unbiased estimate of the local abundance of species. I think that the areas that are moe accessible will have more occurrences and I don’t think that this can be reliably associated with the abundance of the species in the area.

**Comment by phylojenie** [comment]
@superornitho I see your point! I suppose we could look and see if the ratio of species to species scales up to match the occurrence records? For example, if we could 5 butterfly fish and 1 eagle ray, will the occurrence data show 5 butterfly fish observations for every 1 eagle ray? 

_However_ since we have decided to use range maps rather than occurrence data, that point is somewhat obsolete...so perhaps we should just record presence/absence of a species at each location? 1 for present and 0 for absent?
